massgen/subagent/manager.py

```
@@ -818,6 +821,254 @@ async def _execute_with_orchestrator(
+                    log_path=str(log_dir) if log_dir else None,
+                    warning=context_warning,
+                )
+
+            # Update state
+            if result.success:
+                state.status = "completed"
+            else:
+                state.status = "failed"
+            state.result = result
+
+            return result
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)